patterns/2-structured/transparent-cross-team-decision-making-using-rfcs.md

```diff
@@ -98,6 +98,7 @@ Also for decision making outside of pure software design decisions, transparent
 - **Uber** - According to this blog post by Gergely Orosz: [Scaling Engineering Teams via RFCs: Writing Things Down][uber].
 - **Google Design Docs** - As described in this blog post by Malte Ubl [Design Docs at Google][google]
 - **DAZN** (10/2021) - One way that DAZN makes technical decisions is via RFCs. RFCs are used for decisions that apply to engineering-wide processes only! The RFCs live in a GitHub repository, and technical standards are then gradually adopted within their tools and by their engineers. An RFC can be raised by any engineer, and voted on by all engineers. If upvotes exceed downvotes, the RFC is adopted. It’s worth noting, that the RFC voting process hasn’t yet been “stress-tested” by any contentious decisions. - As described in this blog post by Lou Bichard: [Building A DX Team: Lessons Learned][dazn]
+- **SAP** - As described in the blog post [Cross-Product Architecture: Embracing Conway's Law for Better Software Architecture][sap-cpa], SAP has a mature tool-assisted process for document review across teams. It is primarily used for the review of Architecture Decision Records (ADR) originating from cross-team work done on the Cross-Product Architecture collaboration model. This means that the review process in not easily available for decisions on small projects. Also, the documents are not by rule related to InnerSource projects. That means it is not a universal solution for engineering decision making in the company and also not a 1:1 fit for this pattern, but close enough to be mentioned as known instance.
```

Comment:
```suggestion
- **SAP** (03/2024) - SAP has a mature tool-assisted process for document review across teams. It is primarily used for the review of Architecture Decision Records (ADR) originating from cross-team work done on the Cross-Product Architecture collaboration model. Some noteworthy implementation differences from this pattern: The review process is not easily available for decisions on small projects. Also, the documents are not restricted to InnerSource projects only. - As described in the blog post [Cross-Product Architecture: Embracing Conway's Law for Better Software Architecture][sap-cpa].
```

Proposing a number of changes. In summary:
- moving the link to the end
- adding a date of when we found this known instance (i.e. when you shared it). we don't do this consistently for all Known instances in all patterns yet but it may help us in the future when we try to learn how pattern adoption is spreading through different orgs.
- Removing the qualifying sentence at the end. The pattern does claim to be a universal solution for all engineering decisions. Further all implementations of the pattern have differences to the solution outlined in the pattern. We have not highlighted this in other Known Instances either.

Hope my explanations help and you like the changes.

Reply:
@dellagustin-sap I am going ahead with the changes to your PR that I proposed above.

I am currently doing some cleanup in our repo, therefore I am trying to get relatively small PRs like this one merged.

If you don't agree with my reformatting of your text, feel free to send a new PR of course :)

Reply:
Hello @spier , sorry for losing track of this and thank you for following it up, the changes look good to me.